client/src/pages/Profile.js

```diff
@@ -1,84 +1,93 @@
+/** @format */
+
 import React from "react";
 import useStyles from "../styles/Profile";
 import { NavLink, useRouteMatch, Switch, Route } from "react-router-dom";
 import Notifications from "./Notifications";
 import FavoriteDestination from "./FavoriteDestinantions";
 import AccountSettings from "./AccountSettings";
 import { Avatar, Drawer, Typography, Button, Grid } from "@material-ui/core";
+import { useDispatchContext } from "../context";
 
 function Profile() {
-  const { path } = useRouteMatch();
-
-  const classes = useStyles();
-  const mockUser = {
-    avatar:
-      "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTdX7wWCMOvGYD6_4-MthVKf-DjjgLF_GqQzg&usqp=CAU",
-    name: "Devin Jones",
-    email: "DevinJones@gmail.com",
-  };
+	const { path } = useRouteMatch();
+	const dispatch = useDispatchContext();
+	const classes = useStyles();
+	const mockUser = {
+		avatar:
+			"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTdX7wWCMOvGYD6_4-MthVKf-DjjgLF_GqQzg&usqp=CAU",
+		name: "Devin Jones",
+		email: "DevinJones@gmail.com",
+	};
+	const handleLogout = async () => {
+		await fetch(`api/users/logout`);
+    dispatch({ type: "LOG_OUT" });
```

Comment:
Indentation 🙂

Reply:
Strange. I just checked lines 22 -24 and it has proper indentation.

Reply:
Weird